### DISCUSSION TITLE
I have this in `conftest.py`

### DISCUSSION THREAD

**Discussion by BeyondEvil** [pytest-dev/pytest#11030] [Q&A]
I have this in `conftest.py`

```
def pytest_addoption(parser):
    parser.addoption(
        "--cmdopt", action="store", default="value"
    )

@pytest.fixture
def cmdopt(request):
    return request.config.getoption("--cmdopt")
```

What is the best way to read `cmdopt` value inside `pytest_runtest_makereport` function?

_Originally posted by @harmin-parra in https://github.com/pytest-dev/pytest-html/discussions/680_

**Comment by nicoddemus** [comment]
Hi @BeyondEvil ,

`item.config.getoption` should work.

https://docs.pytest.org/en/stable/reference/reference.html#pytest.hookspec.pytest_runtest_makereport

https://docs.pytest.org/en/stable/reference/reference.html#pytest.Item.config

**Comment by BeyondEvil** [comment]
Ping @harmin-parra

**Comment by The-Compiler** [comment]
Converted this to a discussion, since this doesn't seem like an issue.

**Reply by BeyondEvil** [reply]
Thank you.

I think this is a "flaw" in GitHub as the original was a discussion.
